lib/marked.js

```diff
@@ -1516,6 +1511,24 @@ function splitCells(tableRow, count) {
   return cells;
 }
 
+function lineMatch(str, reg, some) {
```

Comment:
What does `some` represent?

Reply:
if it should match some lines or every line.

I could remove it. it isn't really needed here.